Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -51,8 +46,9 @@ namespace ROS2
             { // We have reached the top level, there is no parent entity so there can be no parent ROS2Frame
                 return nullptr;
             }
-
-            const AZ::Entity* parentEntity = AzToolsFramework::GetEntityById(parentEntityId);
+            AZ::Entity* parentEntity;
```

Comment:
minor: initialize to nullptr
```suggestion
            AZ::Entity* parentEntity = nullptr;
```